taskfile/reader.go

```diff
@@ -244,10 +247,14 @@ func (r *Reader) readNode(node Node) (*ast.Taskfile, error) {
 				// If there is a cached hash, but it doesn't match the expected hash, prompt the user to continue
 				prompt = fmt.Sprintf(taskfileChangedPrompt, node.Location())
 			}
+
 			if prompt != "" {
+				r.promptMutex.Lock()
 				if err := r.logger.Prompt(logger.Yellow, prompt, "n", "y", "yes"); err != nil {
+					r.promptMutex.Unlock()
```

Comment:
I thought about using defer, but the return is just after this. Not sure if it's more clear using defer or not

Reply:
I like your arguments, I've tested it and it works

Thanks a lot for your reviews :heart: